.storybook/Setup.ts

```diff
@@ -1,30 +1,46 @@
-import { PerspectiveCamera, Scene, WebGLRenderer } from 'three'
+import { ColorManagement, PerspectiveCamera, Scene, sRGBEncoding, WebGLRenderer } from 'three'
+
+declare global {
+  interface Window {
+    root: HTMLDivElement
+    canvas: HTMLCanvasElement
+    context: WebGL2RenderingContext
+  }
+  const root: HTMLDivElement
+  const canvas: HTMLCanvasElement
+  const context: WebGL2RenderingContext
+}
+
+window.canvas = root.appendChild(document.createElement('canvas'))
+window.context = canvas.getContext('webgl2')!
+
+if ('enabled' in ColorManagement) ColorManagement.enabled = true
+else ColorManagement.legacyMode = false
```

Comment:
This can be simplified to `ColorManagement.enabled = true` by upgrading three.js to r150. Might need to be moved to `preview.ts` or higher up to also catch global colors.